Api/Client.php

```diff
@@ -310,6 +307,72 @@ private function removeLastLineFromFiles($files = null, $noOfLinesToRemove = 1)
 
     /**
      *
+     * @param array $lines
+     * @param int   $noOfLinesToRemove
+     *
+     * @return $lines
+     */
+    private function removeLastLine(array $lines, $noOfLinesToRemove = 1)
+    {
+        $lastLine = count($lines) - $noOfLinesToRemove;
+        unset($lines[$lastLine]);
+        return $lines;
+    }
+
+    /**
+     * @param array $lines
+     * @param string $separator
+     * @param string $enclosure
+     *
+     * @return array
+     */
+    private function fixDate(array $lines, $separator = ',', $enclosure = '"')
+    {
+        foreach ($lines as $key => $line) {
+            $columns = str_getcsv($line, $separator );
+            $isChanged = false;
+            foreach ($columns as $columnKey => $column)
+            {
+                if (preg_match('/^[0-9]{1,2}\/[0-9]{1,2}\/[0-9]{4}$/', $column))
```

Comment:
Good to use this regex, keep doing ;)  But when using file_get_csv you can be more specific when you use a specific column